### PR TITLE
Fix test code according to updated perltidyrc in os-autoinst + compile error

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
@foursixnine @richiejp your urgent feedback would be appreciated here to fix tests

File: lib/hacluster.pm [hunk 1]
```diff
@@ -311,7 +311,7 @@ sub get_lun {
 sub pre_run_hook {
     my ($self) = @_;
 
-    $prev_console = $testapi::selected_console;
+    $prev_console = $autotest::selected_console;
```
Comment (Soulofdestiny) [inline lib/hacluster.pm]: is this substitution really related to perltidy?

Comment (foursixnine) [reply lib/hacluster.pm]: @Soulofdestiny it's related to https://github.com/os-autoinst/os-autoinst/pull/985#discussion_r203073837.

Comment (foursixnine) [reply lib/hacluster.pm]: I recommend to use: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5412/files instead :)

**Comment by Soulofdestiny** [comment]
LGTM

File: lib/hacluster.pm (hunk 1 above)
Comment (ldevulder) [reply lib/hacluster.pm]: Yes should be track in another PR, and also SLES4SAP code needs to be change as well.

I will revert this part, as Qemu changes are not merged yet on o.s.d.
